docs/review_checklist.md

```diff
@@ -0,0 +1,87 @@
+- [ ] Check if there are any TODOs which are not linked to an issue of the
+      zeitgeist repository like this: `TODO(#999)`.
+- [ ] Does the PR link relevant issues and contain a detailed description?
+- [ ] If the PR changes the business logic, does it add the corresponding
+      labels?
+- [ ] Are the doc strings up to date?
+- [ ] If the PR adds or changes extrinsics or functions used by extrinsics:
+  - [ ] Is the _Weight_ section in the documentation up to date?
+  - [ ] Are the benchmarks up to date?
+  - [ ] Do the call filters need to be adjusted (forbidden scoring rules,
+        dispute mechanisms)?
+  - [ ] Do the extrinsics emit all the required events (see [Events](#events)
+        below)?
+- [ ] Is the module `README.md` up to date?
+- [ ] Is [docs.zeitgeist.pm] up to date?
+- [ ] Is `docs/changelog_for_devs.md` up to date, specifically:
+  - [ ] Are changes relevant to the Frontend Team (extrinsics changed, new
+        functions) mentioned here?
+  - [ ] Are all new events explained so they can easily be integrated into the
+        indexer?
+  - [ ] Are breaking changes marked as such?
+  - [ ] Is the file formatted with `prettier -w docs/changelog_for_devs.md`?
+- Sanity tests:
+  - [ ] The local temporary development node produces blocks:
+        `cargo run --profile=production -- --tmp`.
+  - [ ] The node syncs with Zeitgeist and Battery Station:
+        `cargo run --profile=production --features=parachain`,
+        `cargo run --profile=production --features=parachain -- --chain=battery_station`.
+  - [ ] `try-runtime` passes on Zeitgeist and Battery Station.
+- [ ] Code quality:
+  - [ ] Are there compiler warnings which can easily be avoided?
+  - [ ] Is integer arithmetic saturated/checked and are all panickers removed?
+  - [ ] Is code contained in macro invocations (benchmarks,
+        `runtime/common/src/lib.rs`, `decl_runtime_apis!`) correctly formatted?
+  - [ ] Are all `*.toml` files formatted with `taplo` (run
+        `taplo format --check`)?
+  - [ ] Are all copyright notices up to date?
+  - [ ] Are enums sorted alphabetically, except for enums used in storage (to
+        prevent migrations) or errors and events?
+- [ ] If an action is required by the Frontend Team, add an issue to
+      zeitgeistpm/ui.
+- [ ] If the PR adds a new pallet, is the pallet added to the benchmark
+      configuration in `scripts/`?
+- [ ] If you are changing/removing configuration values, storage items: Do the
+      changes require a storage migration?
+- [ ] If you are adding/changing configuration values on the mainnet: Have the
+      implications been discussed with the product owners?
```

Comment:
Adjusted two things here beside the conversion to an action:
1. Removed the constraint "mainnet only", because changing values can also lead to undesired issues on the testnet
2. Replaced project owners by code owners.

```suggestion
- [ ] In case configuration values changed: The
      implications have been discussed with the [code owners](https://github.com/zeitgeistpm/zeitgeist/blob/main/CODEOWNERS).
```